packages/lib-classifier/src/store/SubjectGroup/SubjectGroup.js

```diff
@@ -2,76 +2,24 @@ import { getRoot, tryReference, types } from 'mobx-state-tree'
 import Resource from '../Resource'
 import subjectViewers from '@helpers/subjectViewers'
 import SingleImageSubject from '../SingleImageSubject'
-
-function validateSubjects (subjects) {
-  return subjects.length > 1
-}
+import Subject from '../Subject'
 
 const SubjectGroup = types
   .model('SubjectGroup', {
-    already_seen: types.optional(types.boolean, false),
-    favorite: types.optional(types.boolean, false),
-    finished_workflow: types.optional(types.boolean, false),
-    metadata: types.frozen({}),
-    retired: types.optional(types.boolean, false),
-    selected_at: types.maybe(types.string),
-    selection_state: types.maybe(types.string),
-    shouldDiscuss: types.maybe(types.frozen()),
-    subjects: types.refinement(types.array(SingleImageSubject), validateSubjects),
-    user_has_finished_workflow: types.optional(types.boolean, false),
+    subjectIds: types.array(types.string),
```

Comment:
👍 So a subject group is a Subject model with an added `subjectIDs` array?

Reply:
Yup, that's correct! With the current implementation of the API, there's only a very small difference between a 'standard' Subject and a 'group' Subject.